### ISSUE TITLE
Deployed and Ready to deploy display the same data

### ISSUE THREAD

**Issue by mthimkhulusd** [snipe/snipe-it#1217]
The "Deployed" link displays only assets that were checked out to specific users. I expected it should also display assets that were deployed to other places not only users. The "Ready to deploy" link displays both "Ready to deploy" assets and also "Deployed" assets. I think there's a mistake with that. Please assist.
##

**Comment by snipe** [comment]
I'm not able to reproduce this locally or on the demo. Can you reproduce on the demo?

**Comment by snipe** [comment]
There is no ability to check out assets to locations ("other places") at this point, so I'm not sure what you're expecting to happen here.

**Comment by mthimkhulusd** [comment]
Hi

Please view the below print screen. I expected that the screen should only
show "Ready to Deploy Assets" only, not the "Deployed" assets. Please
assist me with that.


[image: Inline image 1]

Thank you

On Thu, Oct 1, 2015 at 7:20 PM, snipe <notifications@github.com> wrote:

> There is no ability to check out assets to locations ("other places") at
> this point, so I'm not sure what you're expecting to happen here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/snipe/snipe-it/issues/1217#issuecomment-144791868>.
>



-- 
mthimkhulu

**Comment by mthimkhulusd** [comment]
![error](https://cloud.githubusercontent.com/assets/10958836/10307597/76b1748a-6c30-11e5-9009-22147310a747.jpg)

**Comment by snipe** [comment]
Can you show me a screenshot if your status labels page?

**Comment by mthimkhulusd** [comment]
![error](https://cloud.githubusercontent.com/assets/10958836/10336055/37b7cbc8-6cf6-11e5-8c02-2ca41f230d0c.jpg)

**Comment by mthimkhulusd** [comment]
Please view the attached screenshot.

**Comment by PropTim** [comment]
I think she meant this page: 
![cat4](https://cloud.githubusercontent.com/assets/15049080/10485243/06dcf11e-7281-11e5-846f-f24fc997f5c2.PNG)

It's in the top right under [admin] -> [status labels]

**Comment by snipe** [comment]
@mthimkhulusd Any update on this?

**Comment by RTITUser** [comment]
I am having the same issue here, if I am understanding that mthimkhulusd is having the issue of selecting Deployed from the Nav Bar on the left and items with a status other than Deployed are showing. My issue is that I have assigned an asset the status of "Un-deployable" yet when I do a search from the Nav bar for Un-Deployable assets I get the no matching records found message. Now if I do a search for "Ready-to-Deploy" assets there I find the Un-deployable asset in the same list?? Or with my created label of "Awaiting Re-Imaging" no files come up when I search for these, which has been added to the Nav Bar, but if I search on the asset tags the systems show. Trying to understand what may be causing this issue, or what it is that I am doing wrong.
![statuslabelissue1](https://cloud.githubusercontent.com/assets/22746746/20408901/b2e1e4d6-acd4-11e6-8734-e31fa027934c.png)
![statuslabelissue2](https://cloud.githubusercontent.com/assets/22746746/20408902/b2e20560-acd4-11e6-9cd3-db2c19f8c2da.png)
![statuslabelissue3](https://cloud.githubusercontent.com/assets/22746746/20408900/b2e1bdd0-acd4-11e6-8209-790293f37893.png)
![statuslabelissue4](https://cloud.githubusercontent.com/assets/22746746/20408899/b2e19666-acd4-11e6-8d52-1a3ffc668938.png)
